rocketpool-cli/wallet/commands.go

```diff
@@ -9,6 +12,19 @@ import (
 
 // Register commands
 func RegisterCommands(app *cli.App, name string, aliases []string) {
+    var stdoutIsTTY bool = false
```

Comment:
This is close - can you move all of this into `rocketpool-cli/wallet/export.go` like you've done with the stuff in `rocketpool-cli/wallet/init.go`? That should be where the checking logic occurs, not in the command registration bit. Thank you!